src/bots/discord.ts

```diff
@@ -208,7 +216,9 @@ export default class DiscordBot extends BotClient {
     } else if (discordChannel instanceof TextChannel) {
       // Check for the permissions
       const discordUser = discordChannel.members.get(user.id);
-      canEmbed = discordChannel.permissionsFor(discordUser).has('EMBED_LINKS');
+      canEmbed = discordUser
+        ? discordChannel.permissionsFor(discordUser)?.has('EMBED_LINKS') ?? false
+        : false;
```

Comment:
This can look better if the `permissionsFor()` function returns false or undefined when `discordUser` is undefined